python/Ganga/Utility/execute.py

```diff
@@ -261,7 +276,9 @@ def execute(command,
             logger.error("Command: %s" % command)
             logger.error("stdout: %s" % stdout)
             logger.error("stderr: %s" % stderr)
-            raise Exception("Missing update env after running command")
+            # Too specific? Hopefully we'll never know
+            from Ganga.Core.exceptions import RuntimeException
```

Comment:
1. Any import like this should be at the top of the file.
2. This exception does not exist. The built-in Python exception `RuntimeError` should be used.